build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     ext.kotlin_ktx_version = '1.2.0'
     ext.wordPressUtilsVersion = '1.30.3-beta.3'
     ext.detektVersion = '1.15.0'
-    ext.gutenbergMobileVersion = 'v1.50.1'
+    ext.gutenbergMobileVersion = '3116-7fed403b4e1633178004f7dfde74db83c0454fcd'
```

Comment:
Should that `3116` be `3371` instead, to point to https://github.com/wordpress-mobile/gutenberg-mobile/pull/3371?

Reply:
🤦‍♀️

Yes, thank you so much for pointing that out!